### PR TITLE
Add missing tests

### PR DESCRIPTION
**Pull request by XaosSintez**
We have many entities untested and it causes overall coverage at 76%, which is really low.
This PR adds some missed tests and fixes found issues (for example, some code was not compatible with PHP 5.5).
This PR increases coverage to 83.5%

I think i will add some other tests in separated PR (to make review easier).

### CODE REVIEW

File: src/Entities/PaymentRetryInstructions/PaymentInstruction.php [hunk 1]
```diff
@@ -28,7 +28,7 @@
     const PARTIAL = 'partial';
     const DISCOUNT = 'discount';
 
-    const AVAILABLE_METHODS = [
+    private static $availableMethods = [
```
Comment (slavcodev) [inline src/Entities/PaymentRetryInstructions/PaymentInstruction.php]: Why this?

Comment (XaosSintez) [reply src/Entities/PaymentRetryInstructions/PaymentInstruction.php]: @slavcodev It is incompatible with PHP 5.5:
http://php.net/manual/en/migration56.new-features.php

Comment (slavcodev) [reply src/Entities/PaymentRetryInstructions/PaymentInstruction.php]: Oh, we supports 5.5 here. I think we can check API log and check the minimal PHP version used by our clients.

Comment (XaosSintez) [reply src/Entities/PaymentRetryInstructions/PaymentInstruction.php]: Sure. But it should be separated PR and (probably) new SDK version. For now i think it is correct to support minimum declared PHP version compatibility

Comment (slavcodev) [reply src/Entities/PaymentRetryInstructions/PaymentInstruction.php]: Sure.

**Pull request merged**